### ISSUE TITLE
7Span: 3 Stage: Incipit Search Tool (to be rebuilt)

### ISSUE THREAD

**Issue by elambrinaki** [PrincetonPEMM/pemm-issues#51]
The Incipit Search Tool used to look this, before it was taken down. This is a screenshot of a particular search. You can see the Macomber Story IDS on the left, the most important information. Then, you can see the matching words appear in orange. The manuscript name, with the folio on which the incipit appeared is at the bottom of the result. We then have a bar on how good the result is. There were some things we wanted to appear differently (see below), but this is to give you an idea.
![old-incipit-tool](https://user-images.githubusercontent.com/56000106/211421266-e26ea572-20c9-4bdb-b3ce-8108a08addc8.jpg)

**Comment by elambrinaki** [comment]
- [ ] When a story instance record has Scan Start and no Folio Start data, use Scan Start with the manuscript name (current rule is to show the Folio Start, so when a record has not a folio but a scan start, this data is not shown). If both Folio Start and Scan Start are present, use Folio Start.
- [ ] Add the manuscript date after the manuscript name and folio number

**Comment by WendyLBelcher** [comment]
We made some slight changes to CDH wording, headers, indenting. One change, in terms of wording is, instead of "20 results from 3186 results" the wording should be something like "3186 matches for the words you typed" and moved it under the search bar. We talked about putting "excellent match" or "poor match" for the results but decided against that. We will use just the bar and number (although, not with so many decimal points). Also, do we need it to have 3,186 results? Maybe we should have a cut off? Although, if you type in Qemer, it might not show results as it is only one word out of twenty, so never mind
